tests/integration/cli/test_accounts.py

```diff
@@ -67,6 +67,20 @@ def test_import_valid_private_key(ape_cli, runner, temp_account, temp_keyfile_pa
     assert temp_keyfile_path.is_file()
 
 
+@run_once
+def test_import_alias_too_long(ape_cli, runner):
+    # Attempt using private key as the alias.
+    key_alias = f"0x{PRIVATE_KEY}"
```

Comment:
technically 66 characters, but I think it works either way

Reply:
i changed it to 42 characters and also made it so hex is not allowed... so your alias has to be shorter than an address rather than a private key... idk, seems ok, if you run into these problems you are probably fudging something up anyway i would think.

Reply:
I was picturing using an s3 bucket for storing chaosnet bot keyfiles, and using a 3 word alias to ensure that it's hard for any random person to guess the keyfile's name. If we needed to increase entropy there we'd run into these limits